src/components/case_manager/PersonTable.tsx

```diff
@@ -110,28 +88,26 @@ function EnhancedTableHead<Column extends HeadCell>({
 }
 
 interface EnhancedTableToolbarProps {
-  numSelected: number;
   totalRows: number;
   type: "Student" | "Staff";
   onOpenInput: () => void;
+  searchParam: string;
```

Comment:
a little confused what this is, if I'm understanding correctly maybe 
```suggestion
  defaultSearchValue: string;
```
is more appropriate?

Reply:
I wouldn't say its a default. Maybe just searchValue or inputSearchValue?

Reply:
it looks like this component controls the state of the value and it just is provided as the initial state, which is why I suggested naming it `default*`

but could be misunderstanding

Reply:
The state value is defined on line 237 as an empty string. It is passed down to this component, where it can be changed in the TextField from 135-148. The user inputs the search parameter which will then filter the list.

Reply:
🤦 

probably oos for this PR, but maybe splitting up the components into separate files would be good

Reply:
I had that thought as well. But none of the other components get reused outside of the table, which is why I liked leaving them in the same file.